filebeat/hooks/init

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+exec 2>&1
+
+mkdir "{{pkg.svc_data_path}}/logs"
```

Comment:
Would `mkdir -p` be better here?

Reply:
![tenor-23212957](https://user-images.githubusercontent.com/24568/42006762-c23d222c-7ab6-11e8-844b-1ff208f7a6c1.gif)

Not a bad suggestion, added `-p` 👍